tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -757,6 +757,8 @@ def test_get_observation_pairs() -> None:
     def objective(trial: Trial) -> float:
 
         x = trial.suggest_int("x", 5, 5)
+        z = trial.suggest_categorical("z", [None])
+        z  # Use z here
         if trial.number == 0:
             return x
```

Comment:
I think the first line is a bit confusing. For example, how about changing the following branch?
```suggestion
        if trial.number == 0:
            return x * int(z == None)
```

Reply:
Thanks, I applied this change.